frontend/src/sass/base/_stat-cards.scss

```diff
@@ -0,0 +1,63 @@
+@import '~base/sass/base/variables';
+@import '~base/sass/base/functions';
+@import '~base/sass/base/grid';
+
+.stat-card {
+  border: calcRem(1) solid #efefef;
+  background-color: #fff;
+  box-shadow: 0 calcRem(2) calcRem(4) 0 rgba(0,0,0,0.25);
```

Comment:
In general, do we want to define a color scheme and then use the color vars wherever we're hardcoding color values?